lib/appsignal/error_handler.ex

```diff
@@ -32,12 +32,12 @@ defmodule Appsignal.ErrorHandler do
   def handle_event(event, state) do
     state =
      case match_event(event) do
-        {origin, reason, message, stack, metadata} ->
+        {origin, reason, message, stack, metadata, conn} ->
```

Comment:
Great, only I think that match_event always returns `%{}` for the metadata. So it could be left out entirely.